crates/ruff/src/rules/flake8_bugbear/rules/mutable_argument_default.rs

```diff
@@ -50,14 +57,26 @@ use crate::checkers::ast::Checker;
 pub struct MutableArgumentDefault;
 
 impl Violation for MutableArgumentDefault {
+    const AUTOFIX: AutofixKind = AutofixKind::Sometimes;
+
     #[derive_message_formats]
     fn message(&self) -> String {
-        format!("Do not use mutable data structures for argument defaults")
+        format!("Replace mutable data structure with `None` in argument default and replace it with data structure inside the function if still `None`")
+    }
+    fn autofix_title(&self) -> Option<String> {
+        Some(format!(
+            "Do not use mutable data structures for argument defaults"
+        ))
```

Comment:
Should this be reversed as in the new message (the one with "Replace...") would be the autofix title?

Reply:
Yeah that was the original until: https://github.com/astral-sh/ruff/pull/4880#discussion_r1257857815 but that was more of a length argument. Agree that the phrasing now is potentially confusing. Can find a way to get the best of both worlds.

Reply:
Interesting! I think editors would be using the violation message to display instead of the autofix title and I believe that's the top level key in the diagnostic data sent by the LSP server as well. \cc @MichaReiser